src/test/java/com/elasticbox/jenkins/tests/PullRequestTestBase.java

```diff
@@ -197,6 +198,7 @@ private GitHub createGitHubConnection(String githubEndpoint, String githubUser,
         if (githubEndpoint.equals(publicGithubAddress)) {
             github = GitHub.connect(githubUser, githubToken);
         } else {
+            // Deprecated Use with caution. Login with password is not a preferred method. Hay que deidir si quitarlo
```

Comment:
Please, remove Spanish portion of comment and create a Jira ticket to address this change